src/sentry/services/hybrid_cloud/__init__.py

```diff
@@ -154,8 +163,15 @@ def close(self):
         pass
 
     def get_organizations(
-        self, user_id: Optional[id], scope: Optional[str], only_visible: bool
+        self, user_id: int, scope: Optional[str], only_visible: bool
     ) -> List[ApiOrganization]:
+        organizations = self._query_organizations(user_id, scope, only_visible)
```

Comment:
Split this up into two pieces: one piece queries the organizations via the ORM and the other then serializes it down.